packages/workflow/package.json

```diff
@@ -29,8 +29,8 @@
     "typecheck": "tsc",
     "build": "tsc -p tsconfig.build.json",
     "format": "prettier --write . --ignore-path ../../.prettierignore",
-    "lint": "eslint . --quiet",
-    "lintfix": "eslint . --fix",
+    "lint": "eslint . --quiet --report-unused-disable-directives",
+    "lintfix": "eslint . --fix --report-unused-disable-directives",
```

Comment:
we should revert these

Reply:
Please see [69fb7f8](https://github.com/n8n-io/n8n/pull/7537/commits/69fb7f86250587a99901817545cfab89034f7969), I probably realized while you were reviewing

Reply:
I mean, you missed to remove the flag from this one `package.json`.